open-company/open-company-web
# FoC attachments update

Card: https://trello.com/c/RUgAPnyT

To test:
- take a post w/o attachments
- [ ] do you NOT see any mention of attachments in FoC?
- add an attachment to it
- [ ] FoC says 1 Attachment (no final s)?
- [ ] hover on it, do you see it?
- add another attachments
- [ ] FoC has right count and now the final s?
- hover on it
- click an attachment
- [ ] did it NOT open the post?
- [ ] did it open the attachment?
